RecoHGCal/TICL/python/ticl_iterations.py

```diff
@@ -19,6 +20,8 @@
 def TICL_iterations_withReco(process):
```

Comment:
this file should eventually be replaced by parts that do not know about a process:
- a file with tasks defined to do the task
- a RecoHGCal/Configuration/python/RECOHGCal_EventContent_cff.py with all appropriate keep statements